pkgs/development/python-modules/mido/default.nix

```diff
@@ -30,6 +30,11 @@ buildPythonPackage rec {
     })
   ];
 
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "packaging~=23.1" "packaging"
+  '';
```

Comment:
Use `pythonRelaxDepsHook` in `nativeBuildInputs` and `pythonRelaxDeps = [ "packaging" ]` instead.